sim/constants.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Simulator constants
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * @license MIT
+ */
+
+export enum MoveEffectiveness {
```

Comment:
Why a `constants.ts`? This can just be in `pokemon.ts`, no?

Reply:
To facilitate modding, and with future refactors of magic numbers in mind.

Reply:
How does it help with those (sorry, maybe you could elaborate with an example? I'm better with examples :P)?

I'm not saying having a `constants.ts` file is wrong *a priori*, it just seems out of place currently.

Reply:
Actually, there is a better reason: it's now going to be imported by ``battle.ts`` too (WIP) :p